rviz_default_plugins/package.xml

```diff
@@ -35,6 +35,7 @@
   <test_depend>ament_cmake_gmock</test_depend>
   <test_depend>ament_cmake_lint_cmake</test_depend>
   <test_depend>ament_cmake_uncrustify</test_depend>
+  <test_depend>dummy_robot_bringup</test_depend>
```

Comment:
I'm hesitant about this, since I don't expect `dummy_robot_bringup` to exist forever. I'd rather have it self contained in the rviz repository, though I understand that makes it a lot harder.

I'll take it as-is, but please add a TODO to use something less volatile in the future.

Reply:
I agree - that's why we added our own urdf file for testing. This is a leftover and we'll remove it. Thanks for catching it!